modules/acquia_cms_site_studio/acquia_cms_site_studio.module

```diff
@@ -191,14 +191,14 @@ function acquia_cms_site_studio_content_model_role_presave_alter(RoleInterface &
 }
 
 /**
- * Implements hook_ENTITY_TYPE_insert() to add permissions to role.
+ * Implements hook_ENTITY_TYPE_insert().
  */
 function acquia_cms_site_studio_cohesion_helper_category_insert(EntityInterface $entity) {
   _acquia_cms_site_studio_add_permissions($entity);
 }
 
 /**
- * Implements hook_ENTITY_TYPE_insert() to add permissions to role.
+ * Implements hook_ENTITY_TYPE_insert().
```

Comment:
Can we have description in next line as para instead of removing it completely.

Reply:
Done